### ISSUE TITLE
Monitoring validation logs with Tensorboard

### ISSUE THREAD

**Issue by juanenofbit** [jamesdolezal/slideflow#224]
When monitoring with Tensorboard:
`$ tensorboard --logdir=/project_path/models/00001-outcome-HP0`

I can only view the training logs. Are the validation logs actually being saved with the default options?

I am forcing a validation every 300 batches to have enough logs

```
P.train(
    'mammaprint_status',
    params=hp,
    val_strategy='fixed',
    val_fraction=0.28,
    use_tensorboard=True,
    validate_on_batch=300,
)
```

**Comment by jamesdolezal** [comment]
Thanks for raising this issue. It looks like the core problem is that Tensorflow/Keras changed how tensorboard logging works using the default TensorboardCallback and Model.fit() API. I'm working on an update today that should restore logging - it will be included in the 1.4.3 release scheduled later this week.

**Comment by jamesdolezal** [comment]
Fix has been applied to `master` and published in [1.4.3](https://github.com/jamesdolezal/slideflow/releases/tag/1.4.3).
